db/helpers/botUtils.js

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
The nice thing about separating out these validation fns and exporting them as functions is that you can test them outside of the context of a running db.